src/js/__tests__/contentUtils-test.js

```diff
@@ -142,7 +149,7 @@ describe('contentUtils', () => {
         tagName: 'tagName',
       };
       hasInvalidScripts(fakeElement);
-      expect(FOUND_SCRIPTS.get('version').length).toBe(0);
+      expect(FOUND_SCRIPTS.get(UNINITIALIZED).length).toBe(0);
       expect(window.chrome.runtime.sendMessage.mock.calls.length).toBe(0);
     });
     it('should store any script element direct children', () => {
```

Comment:
Some of these test cases, like this one, are meant to cover to cover the Whatsapp scenario. 

Is setting `setCurrentOrigin('FACEBOOK')` safe here? Should we adjust these specific test cases?

Reply:
It's probably fine now, but the architecture makes it a little odd.
- `storeFoundJS` calls into [updateCurrentState](https://github.com/facebookincubator/meta-code-verify/blob/main/src/js/contentUtils.ts#L278).
- `updateCurrentState` reads the "current origin" value, and I've updated that code such that if you try to access it before it has been initialized the code will throw, thus I'm setting it to **something** so that the tests will pass.

It is safe right now because `window.chrome.runtime.sendMessage` is mocked, so the "current origin" is never used in these tests.